visidata/wrappers.py

```diff
@@ -10,10 +10,10 @@
 option('null_value', None, 'a value to be counted as null', replay=True)
 
 def isNullFunc():
-    return lambda v,nulls=set([None, options.null_value]): any((
-        *(v == n for n in nulls),
-        isinstance(v, TypedWrapper)
-    ))
+    nullv = options.null_value
+    if nullv is None:
+        return lambda v: v is None or isinstance(v, TypedWrapper)
+    return lambda v: v is None or v == nullv or isinstance(v, TypedWrapper)
```

Comment:
I think we need to bind `nullv=nullv` in the lambda, due to Python's lexical scoping?  `nullv` is unique to this purpose now but that would be a frustrating bug to come across later :)

Reply:
True, surprises like that are no fun. Good catch :)